src/graphene-matrix.c

```diff
@@ -746,7 +746,25 @@ graphene_matrix_get_value (const graphene_matrix_t *m,
       return 0.f;
     }
 
-  return graphene_simd4f_get (r, col);
+  switch (col)
```

Comment:
It does have something to do with ARM NEON, though it is a bit obfuscated.  The underlying ARM fast type that graphene_simd4f_get calls into (I forget which one) requires its inputs to be compile time constants.  Passing the "col" argument in was causing the build to fail.

If you still want me to break out into a separate commit I can do that.